Formula/hapi-fhir-cli.rb

```diff
@@ -0,0 +1,154 @@
+class HapiFhirCli < Formula
+  desc "Command-line interface for the HAPI FHIR library"
+  homepage "http://hapifhir.io/doc_cli.html"
+  url "https://github.com/jamesagnew/hapi-fhir/releases/download/v3.3.0/hapi-fhir-3.3.0-cli.tar.bz2"
+  sha256 "8342c78598edd9b6509fff0b9cb1de9b277b97f1537342124d1e78523d887d15"
+
```

Comment:
Needs a `bottle :unneeded` and `depends_on :java => "xxx"`

Reply:
Fixed